src/ncp/ncp_base.hpp

```diff
@@ -588,12 +607,48 @@ class NcpBase
     uint8_t mPreferredRouteId;
 #endif
 
+    uint8_t mCurCommandIID;
+
 #if OPENTHREAD_RADIO || OPENTHREAD_CONFIG_LINK_RAW_ENABLE
+    uint8_t mCurTransmitIID;
```

Comment:
Would be good to rename these as well?

`mCurCommandIID` -> `mCurCommandIid`
`mCurTransmitIID` -> `mCurTransmitIid`

Reply:
Done.